app/_src/gateway/breaking-changes/28x.md

```diff
@@ -7,6 +7,19 @@ Before upgrading, review any configuration or breaking changes in this version a
 
 You may need to adopt different upgrade paths depending on your deployment methods, set of features in use, custom plugins, for example.
 
-### Plugins
+## Deployment
 
-For breaking changes to plugins, see the [{{site.base_gateway}} Changelog](/gateway/changelog/) for your {{site.base_gateway}} version.
\ No newline at end of file
+### Amazon Linux 2022 to 2023 rename
+
+Amazon Linux 2022 artifacts are renamed to Amazon Linux 2023, based on AWS's own renaming.
+
+### Ubuntu 18.04 support removed 
```

Comment:
```suggestion
### Ubuntu 18.04 support removed 

> Patch version: 2.8.4.3
```

Wonder if we can start doing something like this to call out a change that happened in a patch. Easier to see at a glance. Same for the release artifacts above.